jvb/src/main/kotlin/org/jitsi/videobridge/AudioSubscription.kt

```diff
@@ -77,5 +80,6 @@ class AudioSubscription() {
 
     fun onConferenceSourceRemoved(descs: Set<AudioSourceDesc>) {
         wantedSsrcs = wantedSsrcs.subtract(descs.map(AudioSourceDesc::ssrc).toSet())
+        conference.subscribedLocalAudioSources.removeAll(descs.map(AudioSourceDesc::sourceName))
```

Comment:
What if another endpoint is subscribed to the same source? E.g. ep `A` subscribes to source `S`, then `B` subscribes to `S`, then `B` leaves. I think you'll need a richer structure to hold this.